activesupport/test/cache/stores/memory_store_test.rb

```diff
@@ -7,6 +7,8 @@
 class MemoryStoreTest < ActiveSupport::TestCase
   def setup
     @cache = lookup_store(expires_in: 60)
+    @namespace = "test-#{Random.rand(16**32).to_s(16)}"
+    @namespaced_cache = lookup_store(expires_in: 60, namespace: @namespace)
```

Comment:
The Redis and Memcache tests use `@namespace` to prevent tests from interfering with each other.  That isn't a concern for `MemoryStore`, so I think we can omit these instance variables, especially since they are only used for a single test.  (See my suggestion further below.)